core/vtk/ttkMergeTreeAutoencoder/ttkMergeTreeAutoencoderUtils.h

```diff
@@ -0,0 +1,104 @@
+/// \ingroup vtk
+/// \class ttk::ttkMergeTreeAutoencoderUtils
+/// \author Mathieu Pont (mathieu.pont@lip6.fr)
+/// \date 2023.
+///
+/// Utils function for manipulating merge and contour tree classes
```

Comment:
can you please add a (minimal) doxygen documentation (at least a sentence that explains what the functions is doing), for each of the functions implemented in this class?

Reply:
Done in the next commit!

Reply:
Thanks!